yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1183,11 +1194,22 @@ object Client extends Logging {
   private def getUserClasspath(
```

Comment:
(I see the other one calls this; both could be merged into a single method, though.)